homework/index.js

```diff
@@ -16,31 +37,139 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
+  function Contributions(inText) {
+    const url = inText.innerHTML;
+    const infoContainer = document.getElementById('info_container');
+    const infoRight = createAndAppend('div', infoContainer, {
+      class: 'info_right',
+      text: 'Contributions:',
+    });
+    fetchJSON(url, (err, data) => {
+      if (err) {
+        renderError(err);
+        return;
       }
+      data.forEach(element => {
+        const ContributorEl = createAndAppend('a', infoRight, {
```

Comment:
Use camelCase variable and parameter names: `contributorEl`

Reply:
By this "contributorEl" I mean : contributor element. So , it is really a camelCase . But yes, the name appears  confusing. 
for the next times I should be more careful with the names . I have to use one standard way in the whole file.

Reply:
camelCase names should start with a lowercase letter. `ContributorEl` -> `contributorEl`. I don't object to the name itself, although I would probably use a few more letters myself, e.g. `contributorElem`.